src/modules/announcements/pages/Announcements.jsx

```diff
@@ -1,3 +1,53 @@
+import { useEffect, useState } from "react";
+import { useLocation } from "react-router-dom";
+import { SearchForm } from "../../../components/SearchForm/SearchForm";
+import { ANNOUNCEMENTS_LIST } from "../../../constants/announcements";
+
 export const AnnouncementList = () => {
-  return <div>announcement list</div>;
+  const [filteredAnnouncements, setFilteredAnnouncements] = useState([]);
+  const { search } = useLocation();
+
+  const filterAnnouncements = (data) => {
+    const { Phrase, Category, City, Animals } = data;
+    let announcements = [...ANNOUNCEMENTS_LIST];
+    let animalsArray;
+
+    announcements = announcements
```

Comment:
You could move it to the `let announcements = ANNOUNCEMENTS_LIST.filter ...` and make it all in one go. Besides, you don't need here filters. One with three conditions in it would be enough.